CONTRIBUTING.md

```diff
@@ -19,6 +20,12 @@ In this file you can find:
   * [Tools manual pages](#tools-manual-pages)
   * [Tools shell completion](#tools-shell-completion)
 
+## License and SPDX
+
+The project uses `GPLv2+` for `tools/` and `LGPLv2.1+` for everything else. When
```

Comment:
we should actually prefer LGPLv2.1+ even for tools/ since it makes it easier to eventually migrate code to the library.  I'm rewording this while applying.